daemon/algod/api/server/router.go

```diff
@@ -88,44 +90,18 @@ func wrapCtx(ctx lib.ReqContext, handler func(lib.ReqContext, echo.Context)) ech
 	}
 }
 
-func makeAuthRoutes(ctx lib.ReqContext, apiToken string, adminAPIToken string, enableProfiler bool) AuthRoutes {
-	authRoutes := make(AuthRoutes)
-	noAuthRoutes := make(map[echo.Route]echo.HandlerFunc)
-	defaultAuthRoutes := make(map[echo.Route]echo.HandlerFunc)
-	adminAuthRoutes := make(map[echo.Route]echo.HandlerFunc)
-	authRoutes[""] = noAuthRoutes
-	authRoutes[apiToken] = defaultAuthRoutes
-	authRoutes[adminAPIToken] = adminAuthRoutes
-	for _, route := range common.Routes {
-		if route.NoAuth {
-			noAuthRoutes[echo.Route{Method: route.Method, Path: route.Path}] = wrapCtx(ctx, route.HandlerFunc)
-			continue
-		}
-		defaultAuthRoutes[echo.Route{Method: route.Method, Path: route.Path}] = wrapCtx(ctx, route.HandlerFunc)
-		adminAuthRoutes[echo.Route{Method: route.Method, Path: route.Path}] = wrapCtx(ctx, route.HandlerFunc)
+// registerHandler registers a set of Routes to [router]. if [prefix] is not empty, it
+// registers the routes to a new sub-router [prefix]
```

Comment:
is this comment incorrect?

Reply:
It's worded a little strangely but I think it's accurate. I probably pulled this comment from the older version of this file.